libs/database/src/workspace.rs

```diff
@@ -671,6 +672,7 @@ pub async fn select_workspace<'a, E: Executor<'a, Database = Postgres>>(
         database_storage_id,
         owner_uid,
         (SELECT name FROM public.af_user WHERE uid = owner_uid) AS owner_name,
+        (SELECT email FROM public.af_user WHERE uid = owner_uid) AS owner_email,
```

Comment:
since we are using more than 2 fields in `af_user`, it might be better to use a JOIN